x-pack/plugin/sql/qa/server/security/src/test/java/org/elasticsearch/xpack/sql/qa/security/RestSqlSecurityIT.java

```diff
@@ -271,8 +271,6 @@ public void testHijackScrollFails() throws Exception {
 
         createAuditLogAsserter().expectSqlCompositeActionFieldCaps("test_admin", "test")
             .expect(true, SQL_ACTION_NAME, "full_access", empty())
-            // one scroll access denied per shard
-            .expect("access_denied", SQL_ACTION_NAME, "full_access", "default_native", empty(), "InternalScrollSearchRequest")
```

Comment:
This is not actually an access denied for SQL_ACTION_NAME , it is for a scroll search, i.e. :
```
.expect("access_denied", SearchTransportService.QUERY_FETCH_SCROLL_ACTION_NAME, "full_access", "default_native", empty(), "InternalScrollSearchRequest")
```
until now we've erroneously carried over the ACTION_ORIGIN header to the child action, which is audited.